Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -759,6 +759,8 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
 
         auto& frame = m_renderView.frameView().frame();
         bool isMainFrame = isMainFrameCompositor();
+        UNUSED_PARAM(frame);
```

Comment:
These vars are used by the LOG_WITH_STREAM macro, which doesn't to anything when log is disabled. The parameters should be marked as unused if log is disabled.

Reply:
done